### PR TITLE
Ability to set default duration

### PR DESCRIPTION
**Pull request by Nosrac**
I wrote up some code to allow for setting the default duration.  Set by holding down option in the duration list

### CODE REVIEW

File: KeepingYouAwake/Base.lproj/MainMenu.xib [hunk 1]
```diff
@@ -681,51 +681,99 @@
...
                                 </connections>
                             </menuItem>
-                            <menuItem title="Indefinitely" id="VeD-dm-81B">
+                            <menuItem title="Indefinitely" id="5EU-sH-Q12">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <connections>
+                                    <action selector="selectTimeInterval:" target="bOj-Ka-Izn" id="00X-0X-WzP"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Set Default: Indefinite" alternate="YES" id="VeD-dm-81B">
```
Comment (newmarcel) [inline KeepingYouAwake/Base.lproj/MainMenu.xib]: The modifier title does not match the default title:

`Set Default: Indefinitely` would be better.

File: KeepingYouAwake/KYAAppController.m [hunk 2]
```diff
@@ -114,6 +115,17 @@ - (void)showMenu:(id)sender
     [self.statusItem popUpStatusItemMenu:self.menu];
 }
 
+#pragma mark - Default Time Interval
```
Comment (newmarcel) [inline KeepingYouAwake/KYAAppController.m]: Sorry to be nitpicky, can you please add a newline after the #pragma? Thanks! 😅

File: KeepingYouAwake/KYAAppController.m [hunk 3]
```diff
@@ -242,18 +254,25 @@ - (void)terminateTimer
...
-    __weak typeof(self) weakSelf = self;
-    dispatch_async(dispatch_get_main_queue(), ^{
-        NSTimeInterval seconds = (NSTimeInterval)[sender tag];
-        [weakSelf activateTimerWithTimeInterval:seconds];
-    });
+	
+	if (sender.alternate)
+	{
+		[self setDefaultTimeInterval: sender.tag];
+	} else {
```
Comment (newmarcel) [inline KeepingYouAwake/KYAAppController.m]: Can you please move `else` and `{` to separate lines?

*(I know, I have a line-wasting codestyle :wink:)*

**Comment by newmarcel** [comment]
Thanks for investing your time to implement this feature. This is a huge help :+1: 

I added some comments for minor issues but all in all it's ready to merge.

**Comment by Nosrac** [comment]
No problem.  I've been wanting the feature for a while now.  After a while I decided I couldn't wait 😉

Let me know if there are any other changes you'd like me to make.

**Comment by newmarcel** [comment]
Merged. Thank you very much! :smile:

**Pull request merged**